maestro-client/src/main/java/maestro/drivers/IOSDriver.kt

```diff
@@ -106,45 +109,42 @@ class IOSDriver(
     }
 
     override fun tap(point: Point) {
-        iosDevice.tap(point.x, point.y).expect {}
+        runDeviceCall { iosDevice.tap(point.x, point.y) }
     }
 
     override fun longPress(point: Point) {
-        iosDevice.longPress(point.x, point.y, 3000).expect {}
+        runDeviceCall { iosDevice.longPress(point.x, point.y, 3000) }
```

Comment:
runDeviceCall is method which is going to be responsible to correctly handle exception which can propagate from `SimctlIOSDevice` and `XCTestIOSDevice`.